packages/flutter/lib/src/material/icon.dart

```diff
@@ -22,24 +24,21 @@ const Map<IconSize, int> _kIconSize = const <IconSize, int>{
   IconSize.s48: 48,
 };
 
-class Icon extends StatelessComponent {
-  Icon({
-    Key key,
-    this.size: IconSize.s24,
-    this.icon: '',
-    this.color,
-    this.colorFilter
-  }) : super(key: key) {
-    assert(size != null);
-    assert(icon != null);
+class DensityQualifier extends BreakpointQualifier {
```

Comment:
Although I'm honestly not sure how useful the change to the material icons class is. We already use a pretty high resolution asset, how valuable is it to store a low res one in addition and sometimes use it?